ecl/eclagent/eclagent.cpp

```diff
@@ -2059,7 +2053,7 @@ void EclAgent::doProcess()
                     case WUStateFailed:
                     case WUStateAborted:
                     case WUStateCompleted:
-                        runWorkunitAnalyser(w, nullptr);
+                        runWorkunitAnalyser(*w, nullptr);
```

Comment:
Here the workunit is already locked.  If the analyseWorkunit locks it again will that cause problems?  It looks like that is safe and efficient.

Reply:
It didn't cause any problems.  However, I'll change the code.